components/Footer.js

```diff
@@ -1,11 +1,11 @@
 import styled from "styled-components";
 import Link from "next/link";
 
-export default function Footer({ buttonlink, buttontext }) {
+export default function Footer({ buttonLink, buttonText }) {
   return (
     <StyledFooter>
       <StyledLinkBorder>
-        <StyledLink href={buttonlink}>{buttontext}</StyledLink>
+        <StyledLink href={buttonLink}>{buttonText}</StyledLink>
```

Comment:
Maybe my first comment was misleading. Please check again: As this is not a button but a link it is confusing to use `buttonLink` and `buttonText` as prop names. I suggested `url` and `linkText` instead.

Reply:
Oopsie, no, your comment was perfectly fine. I think it's the heat 😆